Source/WebCore/Modules/webauthn/AuthenticatorCoordinator.cpp

```diff
@@ -191,6 +207,17 @@ void AuthenticatorCoordinator::create(const Document& document, CredentialCreati
         ASSERT(!exception.message.isNull());
         promise.reject(exception.toException());
     };
+
+    if (m_isCancelling) {
+        m_queuedRequest = [weakThis = WeakPtr { *this }, weakFrame = WeakPtr { *frame }, clientDataJsonHash = WTFMove(clientDataJsonHash), createOptions = WTFMove(createOptions), callback = WTFMove(callback)]() mutable {
+            if (!weakThis || !weakFrame)
+                return;
+            const auto& options = createOptions.publicKey.value();
+            const auto& callerOrigin = weakFrame->document()->securityOrigin();
```

Comment:
Please store in a `Ref`. We need a strong pointer on the stack, per our smart pointer guidelines since you're passing it as argument to another function.